examples/basic-no-config/version.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5"
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = ">= 1.67.0"
```

Comment:
For the examples, we usually try to pick the most basic example and set it to the minimum version of supported provider version, so that we always have one test at the minimum version. This alerts us of a provider/platform change that warrants a change the minimum required version. I think the no-config might be the best example for this, so you could set the version of this example to just "1.67.0".

Reply:
Tested this change locally and it passed. Adding this change to the next push.